vivisect/__init__.py

```diff
@@ -1226,8 +1226,15 @@ def previewCode(self, va, arch=envi.ARCH_DEFAULT):
         '''
         Show the repr of an instruction in the current canvas *before* making it that
         '''
-        op = self.parseOpcode(va, arch)
-        self.vprint("0x%x  (%d bytes)  %s" % (va, len(op), repr(op)))
+        try:
+            op = self.parseOpcode(va, arch)
+            if op == None:
+                self.vprint("0x%x - None")
+            else:
+                self.vprint("0x%x  (%d bytes)  %s" % (va, len(op), repr(op)))
+        except Exception, e:
```

Comment:
`except Exception as e:`

Reply:
doh!  sorry.  wrote this a while ago. :)